doc/writing.md

```diff
@@ -112,8 +112,7 @@ applications like Slack or GitHub):
 :tada:
 ```
 
-Most of the existing emoji are listed on this cheat sheet:
-http://www.emoji-cheat-sheet.com/.
+Refer [this](http://www.emoji-cheat-sheet.com/) emoji cheat sheet.
```

Comment:
Refer **this** emoji cheat sheet seems fine. What do you think? Does it sound too odd?